openapi/components/schemas/component.yaml

```
@@ -0,0 +1,24 @@
+          $ref: ./drop.yaml
+      ducats:
+        type: number
+      imageName:
+        minLength: 1
+        type: string
+      itemCount:
+        type: number
+      masterable:
+        type: boolean
+      primeSellingPrice:
+        type: number
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)